db/migrate/20200521150213_create_storage_services.rb

```diff
@@ -0,0 +1,18 @@
+class CreateStorageServices < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_services do |t|
+      t.string :name
+      t.string :description
+      t.bigint :project_id
+      t.bigint :profile_id
+      t.string :uuid
+      t.bigint :version
+      t.references :parent_service, :type => :bigint, :index => true, :references => :storage_service
+      t.string :capability_values
```

Comment:
What would the value of this look like?  This looks like it might end up being a comma separated mess

Reply:
This is a key value dictionary. This comes from the ems (autosde provider) and is an abstraction/standarization of capabilities across different kinds of storages. In our current first release there is no practical use for it so we will drop it for now. (later releases will relay on these capabilities for provisioning the correct volume)